server/utils/validateEmail.js

```diff
@@ -0,0 +1,6 @@
+const validateEmail = (email) => {
```

Comment:
we can use package for this. There are alot of packages for validation like express validatior, joi etc.